[2022-10-05 13:35:56,878] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: dag_combustivel.pyspark_task scheduled__2016-07-01T00:00:00+00:00 [queued]>
[2022-10-05 13:35:56,891] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: dag_combustivel.pyspark_task scheduled__2016-07-01T00:00:00+00:00 [queued]>
[2022-10-05 13:35:56,892] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-10-05 13:35:56,893] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-10-05 13:35:56,894] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-10-05 13:35:56,913] {taskinstance.py:1264} INFO - Executing <Task(DataprocSubmitJobOperator): pyspark_task> on 2016-07-01 00:00:00+00:00
[2022-10-05 13:35:56,919] {standard_task_runner.py:52} INFO - Started process 12186 to run task
[2022-10-05 13:35:56,923] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag_combustivel', 'pyspark_task', 'scheduled__2016-07-01T00:00:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/dag_combustive.py', '--cfg-path', '/tmp/tmpi0ci_5lh', '--error-file', '/tmp/tmp0js7ricl']
[2022-10-05 13:35:56,924] {standard_task_runner.py:77} INFO - Job 59: Subtask pyspark_task
[2022-10-05 13:35:57,002] {logging_mixin.py:109} INFO - Running <TaskInstance: dag_combustivel.pyspark_task scheduled__2016-07-01T00:00:00+00:00 [running]> on host 1445d51ba90b
[2022-10-05 13:35:57,080] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_combustivel
AIRFLOW_CTX_TASK_ID=pyspark_task
AIRFLOW_CTX_EXECUTION_DATE=2016-07-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2016-07-01T00:00:00+00:00
[2022-10-05 13:35:57,082] {dataproc.py:1878} INFO - Submitting job
[2022-10-05 13:35:58,189] {dataproc.py:1890} INFO - Job 0e78bcec-775f-4eda-942d-bdcda2f09195 submitted successfully.
[2022-10-05 13:35:58,218] {dataproc.py:1903} INFO - Waiting for job 0e78bcec-775f-4eda-942d-bdcda2f09195 to complete
[2022-10-05 13:36:31,514] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/dataproc.py", line 1905, in execute
    job_id=job_id, region=self.region, project_id=self.project_id, timeout=self.wait_timeout
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 433, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/dataproc.py", line 780, in wait_for_job
    raise AirflowException(f'Job failed:\n{job}')
airflow.exceptions.AirflowException: Job failed:
reference {
  project_id: "stack-data-pipeline01"
  job_id: "0e78bcec-775f-4eda-942d-bdcda2f09195"
}
placement {
  cluster_name: "dataproc-cluster-stack-data-pipeline01"
  cluster_uuid: "e897f500-dd3c-4b65-a2ba-450e350e051c"
}
pyspark_job {
  main_python_file_uri: "gs://stack-data-pipeline-combustiveis-brasil-pyspark-code/main.py"
  args: "--path_input"
  args: "gs://ac-data-challenge-combustiveis-brasil-gl-raw/combustiveis-brasil/2016/02/ca-2016-02.csv"
  args: "--path_output"
  args: "gs://ac-data-challenge-combustiveis-brasil-gl-curated/combustiveis-brasil/2016/02/"
  args: "--file_format"
  args: "parquet"
  args: "--bq_dataset"
  args: "data_challenge"
  args: "--table_bq"
  args: "combustiveis_brasil"
  jar_file_uris: "gs://spark-lib/bigquery/spark-bigquery-with-dependencies_2.12-0.23.2.jar"
}
status {
  state: ERROR
  details: "File not found: gs://stack-data-pipeline-combustiveis-brasil-pyspark-code/main.py"
  state_start_time {
    seconds: 1664976988
    nanos: 749656000
  }
}
status_history {
  state: PENDING
  state_start_time {
    seconds: 1664976958
    nanos: 340606000
  }
}
status_history {
  state: SETUP_DONE
  state_start_time {
    seconds: 1664976958
    nanos: 410421000
  }
}
status_history {
  state: RUNNING
  state_start_time {
    seconds: 1664976958
    nanos: 812300000
  }
}
driver_control_files_uri: "gs://dataproc-staging-us-central1-538516841627-vuyoxokq/google-cloud-dataproc-metainfo/e897f500-dd3c-4b65-a2ba-450e350e051c/jobs/0e78bcec-775f-4eda-942d-bdcda2f09195/"
driver_output_resource_uri: "gs://dataproc-staging-us-central1-538516841627-vuyoxokq/google-cloud-dataproc-metainfo/e897f500-dd3c-4b65-a2ba-450e350e051c/jobs/0e78bcec-775f-4eda-942d-bdcda2f09195/driveroutput"
job_uuid: "0e78bcec-775f-4eda-942d-bdcda2f09195"
done: true

[2022-10-05 13:36:31,532] {taskinstance.py:1282} INFO - Marking task as FAILED. dag_id=dag_combustivel, task_id=pyspark_task, execution_date=20160701T000000, start_date=20221005T133556, end_date=20221005T133631
[2022-10-05 13:36:31,551] {standard_task_runner.py:92} ERROR - Failed to execute job 59 for task pyspark_task
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/dataproc.py", line 1905, in execute
    job_id=job_id, region=self.region, project_id=self.project_id, timeout=self.wait_timeout
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 433, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/dataproc.py", line 780, in wait_for_job
    raise AirflowException(f'Job failed:\n{job}')
airflow.exceptions.AirflowException: Job failed:
reference {
  project_id: "stack-data-pipeline01"
  job_id: "0e78bcec-775f-4eda-942d-bdcda2f09195"
}
placement {
  cluster_name: "dataproc-cluster-stack-data-pipeline01"
  cluster_uuid: "e897f500-dd3c-4b65-a2ba-450e350e051c"
}
pyspark_job {
  main_python_file_uri: "gs://stack-data-pipeline-combustiveis-brasil-pyspark-code/main.py"
  args: "--path_input"
  args: "gs://ac-data-challenge-combustiveis-brasil-gl-raw/combustiveis-brasil/2016/02/ca-2016-02.csv"
  args: "--path_output"
  args: "gs://ac-data-challenge-combustiveis-brasil-gl-curated/combustiveis-brasil/2016/02/"
  args: "--file_format"
  args: "parquet"
  args: "--bq_dataset"
  args: "data_challenge"
  args: "--table_bq"
  args: "combustiveis_brasil"
  jar_file_uris: "gs://spark-lib/bigquery/spark-bigquery-with-dependencies_2.12-0.23.2.jar"
}
status {
  state: ERROR
  details: "File not found: gs://stack-data-pipeline-combustiveis-brasil-pyspark-code/main.py"
  state_start_time {
    seconds: 1664976988
    nanos: 749656000
  }
}
status_history {
  state: PENDING
  state_start_time {
    seconds: 1664976958
    nanos: 340606000
  }
}
status_history {
  state: SETUP_DONE
  state_start_time {
    seconds: 1664976958
    nanos: 410421000
  }
}
status_history {
  state: RUNNING
  state_start_time {
    seconds: 1664976958
    nanos: 812300000
  }
}
driver_control_files_uri: "gs://dataproc-staging-us-central1-538516841627-vuyoxokq/google-cloud-dataproc-metainfo/e897f500-dd3c-4b65-a2ba-450e350e051c/jobs/0e78bcec-775f-4eda-942d-bdcda2f09195/"
driver_output_resource_uri: "gs://dataproc-staging-us-central1-538516841627-vuyoxokq/google-cloud-dataproc-metainfo/e897f500-dd3c-4b65-a2ba-450e350e051c/jobs/0e78bcec-775f-4eda-942d-bdcda2f09195/driveroutput"
job_uuid: "0e78bcec-775f-4eda-942d-bdcda2f09195"
done: true

[2022-10-05 13:36:31,611] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-10-05 13:36:31,652] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
